dot-env-example

```diff
@@ -1,2 +1,3 @@
-# Copy this file to .env to expose these environment variables
+PORT=8080
```

Comment:
While troubleshooting, I copied/pasted the contents of dot-env-example in the BDD class lab into our dot-env-example file. It doesn't look like anything broke?